connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/model/v3/Oas30ModelHelper.java

```diff
@@ -80,12 +83,12 @@ public static List<String> getSchemes(Oas30Document openApiDoc) {
                     try {
                         return new URL(serverUrl).getProtocol();
                     } catch (MalformedURLException e) {
-                        LOG.warn(String.format("Unable to determine base path from server URL: %s", serverUrl));
+                        LOG.warn(String.format(NO_URL_ERROR_MESSAGE, serverUrl));
                         return null;
                     }
                 })
         .filter(Objects::nonNull)
-        .collect(Collectors.toList());
+        .toList();
```

Comment:
note: this makes the schemes unmodifiable. don't know if that has any impact on anyone. maybe use `collect(toCollection(ArrayList::new))` just to be sure?